src/components/tasks/card/index.tsx

```diff
@@ -514,62 +514,57 @@ const Card: FC<CardProps> = ({
-                                    }}
-                                    onChange={(e) => {
-                                        handleAssignment(e);
-                                        debounce(
-                                            fetchUsers,
-                                            400
-                                        )(e.target.value);
-                                    }}
-                                    role="button"
-                                    tabIndex={0}
-                                />
-
-                                {isLoadingSuggestions ? (
-                                    <Loader />
-                                ) : (
-                                    showSuggestion && (
-                                        <SuggestionBox
-                                            suggestions={suggestions}
-                                            onSelectAssignee={handleClick}
-                                        />
-                                    )
-                                )}
-                            </div>
-                        )
-                    ) : (
-                        <span className={classNames.cardStrongFont}>
-                            {cardDetails.assignee}
-                        </span>
-                    )}
-                </div>
-            )}
+
+            <div className={classNames.contributor}>
+                <span className={classNames.cardSpecialFont}>
+                    {cardDetails.assignee ? 'Assigned to' : 'Assign to'}
+                </span>
+                <span className={classNames.contributorImage}>
+                    <Image
+                        src={assigneeProfileImageURL}
+                        alt={cardDetails.assignee || DUMMY_NAME}
+                        width={30}
+                        height={30}
+                    />
+                </span>
+                {isEditable
+                    ? isUserAuthorized && (
+                          <div className={classNames.suggestionDiv}>
+                              <input
+                                  data-testid="assignee-input"
+                                  ref={inputRef}
+                                  value={assigneeName}
+                                  className={classNames.cardStrongFont}
+                                  onKeyDown={(e) => {
+                                      handleChange(e, 'assignee');
+                                  }}
+                                  onChange={(e) => {
+                                      handleAssignment(e);
+                                      debounce(fetchUsers, 400)(e.target.value);
+                                  }}
+                                  role="button"
+                                  tabIndex={0}
+                              />
+
+                              {isLoadingSuggestions ? (
+                                  <Loader />
```

Comment:
can we write separate functions and then use them here??

Reply:
This is not the scope of this PR. A separate issue needs to be created and solved for that. There was an immediate requirement for this change to ease converting issue to task .

Reply:
so please create an issue, assign it to you and link the issue to this PR

Reply:
https://github.com/Real-Dev-Squad/website-status/issues/818
Haven't assigned it to myself - don't have the bandwidth as of now to see this.